CONTRIBUTING.md

```diff
@@ -1,54 +1,59 @@
 # CONTRIBUTING
-
-Contributing to the official buyTogether Application
+Contributing to the official buyTogether application
 Welcome! We're so excited that you're considering contributing to the official GitHub page for buyTogether.
-
 ---
-
-## Make changes locally
-
-- Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
-
-- Create a working branch 
-
-- Set up a development environment to begin working on the project all you need to do is 'vagrant up'
-
+## Prerequisites
+- Fork the repo to make your changes without affecting the original project until you're ready to merge.
+- Create a working branch.
+- Set up a development environment to begin working on the project, all you need to do is 'vagrant up'
 - Start with your changes!
-
 ---
-
-## Commit your update
-Commit the changes once you are happy with them. 
-
+## Commiting
+Please use the guide below inorder to follow our commit message convention:
+* [commit convention](https://cbea.ms/git-commit/#seven-rules)
```

Comment:
I think this standard is a good fit. Just note that the commit message for this MR doesn't comply with it either. Specifically, [this point](https://cbea.ms/git-commit/#imperative). The commit message should be something like: `Modify contributing file`.

Reply:
(to be clear, I think you should change this commit message rather than the standard)

Reply:
will change the title